controllers/reconcilers/logging_installation/logging_installation_reconciler.go

```diff
@@ -357,8 +357,10 @@ func (r *Reconciler) createClusterLogForwarderCr(ctx context.Context, cr *v1.Obs
 			clusterLogForwarder.Spec.Inputs = []v14.InputSpec{
 				{
 					Name: "kafka-operator-application-logs",
-					Application: &v14.Application{Namespaces: []string{"redhat-kas-fleetshard-operator", "redhat-managed-kafka-operator",
-						"redhat-kas-fleetshard-operator-qe", "redhat-managed-kafka-operator-qe"}},
+					Application: &v14.Application{Namespaces: []string{"redhat-kas-fleetshard-operator",
+						"redhat-managed-kafka-operator", "redhat-kas-fleetshard-operator-qe",
+						"redhat-managed-kafka-operator-qe", "openshift-logging",
+						"managed-application-services-observability"}},
```

Comment:
We could envision to make these configurable.

Reply:
:+1: